autobuild/windows/autobuild_windowsinstaller_1_prepare.ps1

```diff
@@ -12,6 +12,8 @@ param(
     [string] $BuildOption = ""
 )
 
+# Fail early on all errors
+$ErrorActionPreference = "Stop"
```

Comment:
I considered adding this line at the very top of the script, but this breaks `param()` which has to be the first statement in a script (when used).